src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsEnums.cs

```diff
@@ -214,6 +214,7 @@ public enum EnvChangeType : byte
         public const byte FEATUREEXT_AZURESQLSUPPORT = 0x08;
         public const byte FEATUREEXT_DATACLASSIFICATION = 0x09;
         public const byte FEATUREEXT_UTF8SUPPORT = 0x0A;
+        public const byte FEATUREEXT_AZURESQLDNSCACHING = 0x0B;
```

Comment:
I think we should just name this feature SQLDNSCaching everywhere instead of using Azure in the name.

Reply:
Yeah, I know it's a bit inconsistent here. In the specs from the server team, they use `AZURESQLDNSCACHING` for this new feature ext so I use it here.